graph/impl/src/main/java/org/jboss/windup/graph/GraphContextImpl.java

```diff
@@ -85,6 +87,7 @@ public GraphContextImpl create()
         FileUtils.deleteQuietly(graphDir.toFile());
         TitanGraph titan = initializeTitanGraph();
         initializeTitanIndexes(titan);
+        generateTypeScriptModels();
```

Comment:
Yes, this is the wrong place for the code, IMO. I don't think that is all that important right now. It really does need to just be a standalone tool eventually, though.